### PR TITLE
🧿 User actions for Ajna navigation

### PR DESCRIPTION
**Pull request by piotrkonowrocki**
# 🧿 User actions for Ajna navigation

image
image

#### TODO:
- Add wallet controllers,
- mobile support.
  
## Changes 👷‍♀️

- Created `NavigationOrb` component which is a common component for all user actions and panels in navigation,
- implemented `SwapOrb` and `NotificationsOrb` using `NavigationOrb` so both of them can use the same styling, animation and interaction,
- added "Connect wallet" button to Ajna navigation,
- created `useAccount` hook that contains selected information about user based on global context as a shorthand; for now it's if user is connected and how many positions he has open. That should be further whenever there is opportunity for something to be moved there,
  
## How to test 🧪

- Go to `/ajna` page with `Ajna` feature flag enabled. Without it there should be redirect to homepage,
- test if orbs behavior is similar to those on main landing page (but better 😎),
- test if orbs on main landing page wasn't somehow affected by this changes.

### CODE REVIEW

File: components/navigation/content/MyPositionsLink.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { Spinner } from 'theme-ui'
+
+export function MyPositionsLink() {
+  const { t } = useTranslation()
+  const { amountOfPositions } = useAccount()
+
+  return (
+    <>
+      {t('my-positions')} (
+      {amountOfPositions || (
```
Comment (piekczyk) [inline components/navigation/content/MyPositionsLink.tsx:14]: I think we shouldn't show 0 when user has 0 positions

Comment (piekczyk) [reply components/navigation/content/MyPositionsLink.tsx:14]: image

File: components/navigation/NavigationMenuOrb.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            sx={{
+              position: 'absolute',
+              top: '3px',
+              right: '3px',
+              width: '10px',
+              height: '10px',
+              borderRadius: 'round',
+              bg: 'interactive100',
+              '&::before, &::after': {
+                content: '""',
```
Comment (piekczyk) [inline components/navigation/NavigationMenuOrb.tsx:65]: `content: ''` won't be enough?

**Review by piekczyk — CHANGES_REQUESTED** [review]

File: components/navigation/content/MyPositionsLink.tsx (hunk 1 above)
Comment (piekczyk) [reply components/navigation/content/MyPositionsLink.tsx:14]: on anja infinity loading  
image

Comment (piotrkonowrocki) [reply components/navigation/content/MyPositionsLink.tsx]: Good catch, fixed.

File: components/navigation/NavigationMenuOrb.tsx (hunk 2 above)
Comment (piotrkonowrocki) [reply components/navigation/NavigationMenuOrb.tsx:65]: No, it won't work.
```
content: '',
```
for `theme.ui` is equivalent of not providing any value to `content` property. `'""`' actually transfers to
```
content: ""
```
in css and only then browser treats `content` as non-empty and renders it.

**Review by piekczyk — APPROVED** [review]

**Pull request merged**